experimental/cdf_asset_hierarchy_replication/transformations/asset_hierarchy_replication-load-data_model.sql

```diff
@@ -0,0 +1,13 @@
+select
+  cast(`externalId` as STRING) as externalId,
+  node_reference('{{space}}', `parentExternalId`) as parent,
+  node_reference('{{space}}', '{{root_asset_external_id}}') as root,
+  cast(`name` as STRING) as name,
+  cast(`source` as STRING) as source,
+  cast(`description` as STRING) as description,
+  cast(`labels` as ARRAY < STRING >) as labels,
+  to_json(`metadata`) as metadata
```

Comment:
I think it would be wise to already now show the pattern for how to extract specific keys from the metadata and transform into properties in the view.